amida-tech/saraswati-hedis-results-api
# [SAR-336] Seed data at last.

# Related Tickets

- [SAR-336](https://jira.amida-tech.com/browse/SAR-336)

# How Things Worked (or Didn't) Before This PR

The seed data was out of date.

# How Things Work Now (And How to Test)
Now we have 50 samples to draw from.

To test:

1. Go into `test/seed-data/` to find the files. Open and copy the contents to Postman.
2. Make Post calls to `localhost:4000/api/v1/measures/bulk`. The files are large and it will take a moment.
3. Once done, you can check the results making a Get call to `localhost:4000/api/v1/measures/search?measurementType=<type>`
4. Run `yarn test` to ensure I've broken nothing.

**Notice: ** You probably could submit individual entries on the `develop` branch but you would probably get a "Payload too Large" error. A one-line change to `express` increases file limit sizes to 50 MBs. This is primarily for development purposes only and we may do away with RESTful pushes into the database in production. To be discussed. 

# Readiness


1. [ ] This PR passes all automated tests.
2. [ ] This PR has no linting errors.
3. [ ] This PR's changes to configuration files have been documented in all appropriate places (such as but not limited to README.md), if applicable.
